templates/.env.j2

```diff
@@ -13,10 +13,16 @@ YAGNA_API_URL_PORT="{{ ya_requestor_yagna_api_url_port }}"
 GSB_URL="{{ ya_requestor_gsb_url }}"
 YA_NET_BIND_URL="{{ ya_requestor_net_bind_url }}"
 
+{% if ya_requestor_mumbai_geth_addr %}
+MUMBAI_GETH_ADDR="{{ ya_requestor_mumbai_geth_addr }}"
+{% endif %}
+
```

Comment:
Listing all env variables used by yagna here would be a nightmare to maintain. Therefore only those strictly related to running yagna are listed here. For everything else please use `ya_requestor_extra_env`.

Like this:
`host_vars/your_machine.yml`:
```yml
ya_requestor_extra_env: |
  MUMBAI_GETH_ADDR="whatever"
```

Please remove this.